### PR TITLE
Replaced c = CogniteClient() with client = CogniteClient

### PR DESCRIPTION
**Pull request by andeplane**
## Description
In the Python SDK documentation, we usually have `c = CogniteClient()` as the example to initiate the SDK client. However, shortening this from `client` to `c` has some consequences. One is that it's often challenging to read code with such variable names, in addition to AI being worse at code generation with non descriptive variable names.

In the AI team we work on teaching LLMs how to write code with Python SDK and rely heavily on the SDK documentation where we'd like to use `client = ` instead of `c = ` to get a stronger signal on that variable for the LLM. Human users will likely also benefit from this.

## Checklist:
- [x] Tests added/updated.
- [x] Documentation updated. Documentation is generated from docstrings - these must be updated according to your change.
  If a new method has been added it should be referenced in [cognite.rst](https://github.com/cognitedata/cognite-sdk-python/blob/master/docs/source/cognite.rst) in order to generate docs based on its docstring.
- [x] Changelog updated in [CHANGELOG.md](https://github.com/cognitedata/cognite-sdk-python/blob/master/CHANGELOG.md).
- [x] Version bumped. If triggering a new release is desired, bump the version number in [_version.py](https://github.com/cognitedata/cognite-sdk-python/blob/master/cognite/client/_version.py) and [pyproject.toml](https://github.com/cognitedata/cognite-sdk-python/blob/master/pyproject.toml) per [semantic versioning](https://semver.org/).

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [tool.poetry]
 name = "cognite-sdk"
 
-version = "7.20.1"
+version = "7.20.2"
```
Comment (doctrino) [inline pyproject.toml:4]: Not necessary?

**Review by doctrino — APPROVED** [review]

File: pyproject.toml (hunk 1 above)
Comment (andeplane) [reply pyproject.toml:4]: I'd like the docs to be published for us to pick it up for AI consumption (we use the deployed SDK as documentation source).

**Comment by haakonvt** [comment]
Very nice, please update: `scripts/generate_code_snippets.py`

File: pyproject.toml (hunk 1 above)
Comment (haakonvt) [reply pyproject.toml:4]: Will be updated without version bump

**Comment by andeplane** [comment]
> Very nice, please update: `scripts/generate_code_snippets.py`

I don't understand what this does. Any hints?

**Comment by andeplane** [comment]
> match code to the official documentation based on the links/urls in the docstrings. You can u

What do I need to update? It seems to already cover `client = CogniteClient()`

**Pull request merged**